src/app/common/components/FilterToolbar/FilterControl.tsx

```diff
@@ -9,9 +9,10 @@ import {
 } from './FilterToolbar';
 import SelectFilterControl from './SelectFilterControl';
 import SearchFilterControl from './SearchFilterControl';
+import { IPlan } from '@app/queries/types';
```

Comment:
We also shouldn't have `IPlan` in here, since FilterControl is also supposed to be generic. I know you had to leave for the day, I'll just push a commit to fix it and merge.

Reply:
Yeah exactly, that was not ideal but working!